esp/applications/eclservices/eclservices.yaml

```diff
@@ -36,6 +36,11 @@ eclservices:
         xslt:
         - name: def_file
           ^: "./smc_xslt/def_file.xslt"
+   WsDfs:
+      DisableUppercaseTranslation: false
+      ViewTimeout: 1000
+      EclServer: ''
+      EclServerQueue: ''
```

Comment:
NB: not entirely sure if this section has to be here at all

Reply:
You only need this section if there are service specific config values that you want the user to be able to override.  In that case these will be the defaults.  

Services that exist in different applications can have different defaults for each value.  For example some services exist in both eclwatch (web app and services) and eclservices (services only, currently usually internal to the cluster).

Reply:
There are currently no service configuration option, but perhaps it is best to remove these irrelevant settings and leave WsDfs: as a placeholder/indicator for any future defaults.